polyfills/Element/prototype/dataset/detect.js

```diff
@@ -0,0 +1,6 @@
+(document.documentElement.dataset &&
+	(
+		Object.getOwnPropertyDescriptor(Element.prototype, 'dataset')  &&
+		Object.getOwnPropertyDescriptor(Element.prototype, 'dataset').get
```

Comment:
According to MDN every browser which has dataset also has `Object.getOwnPropertyDescriptor`.
https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/dataset#Browser_compatibility
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/getOwnPropertyDescriptor#Browser_compatibility

Shall I update the detect to fallback to `__getter__` instead?